.circleci/config.yml

```diff
@@ -167,6 +167,8 @@ jobs:
     macos:
       xcode: "12.5.1"
     resource_class: large
+    environment:
```

Comment:
Do you think it will be good idea to have some nightly job to test bundled/ default path for all os's ?

Reply:
Yes, I think it might even be good to have one of the PR jobs use system deps e.g. duplicate the linux job and run one with system and one with bundled to make sure everything works. In the future we could replace the system deps in that job with a conda env.